### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Tchilo**
- set up linters 
- Add person file 
- Add student file 
- Add teacher file 
- fix linter errors

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(*args, classroom: 'Unknown', **kwargs)
+    super(*args, **kwargs)
+    @classroom = classroom
+  end
```
Comment (Dipeshtwis) [inline student.rb]: - [ ] In the `initialize` method argument, the value `"unknown"` should be assigned to `name` and not to the classroom variable.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Tchilo, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (Tchilo) [reply student.rb]: My bad thanks for that 😃.

**Review by mikethreels — APPROVED** [review]
Hi TC, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**